src/main/java/com/zaxxer/nuprocess/linux/ProcessEpoll.java

```diff
@@ -213,12 +220,11 @@ public boolean process()
             else if (ident == stderrFd) {
                linuxProcess.readStderr(NuProcess.BUFFER_CAPACITY, stderrFd);
             }
-         }
-         else if ((events & LibEpoll.EPOLLOUT) != 0) // Room in stdin pipe available to write
-         {
+         } else if ((events & LibEpoll.EPOLLOUT) != 0) { // Room in stdin pipe available to write
```

Comment:
NuProcess code-style is close-curly on a line by itself.

Reply:
I thought I'd reverted all the places I changed this. Will fix